assets/fonts/ooyala-slick-type.svg

```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1">
 <metadata>
-Created by FontForge 20170730 at Sun Dec  3 16:46:33 2017
+Created by FontForge 20170730 at Fri Mar 16 16:00:39 2018
```

Comment:
These icons still have the issue brought up in https://github.com/ooyala/html5-skin/pull/1024. Please check to see if regenerating the icons helps with this problem.